releases/v0.8.0.md

```diff
@@ -0,0 +1,67 @@
+# Release Notes for v0.8.0
+
+Release Date: Oct TBD, 2023
+
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers.
+
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
+and [Glossary](https://github.com/confidential-containers/documentation/wiki/Glossary) pages for a
+definition of the acronyms used in this document.
+
+## What's new
+
+* Confidential Data Hub (CDH) allows users to do in-TEE secret processing.
+  * Supports sealed Kubernetes secrets that can be decrypted by KMS or stored in KBS
+* Support added for multi-KBS model, where one KBS validates evidence using Attestation Service while another provides resource.
+* Enable pulling container images on host which enables image sharing between multiple pods.
```

Comment:
I'm not sure if this worked properly with the current version? See https://github.com/kata-containers/kata-containers/issues/8083

Reply:
Yeah don't think so.